doc/procedures/jira_workflow.rst

```diff
@@ -0,0 +1,86 @@
+Jira workflow
```

Comment:
Maybe rename this "Development Workflow"?

Reply:
Done